rocks.inspectit.releaseplugin/pom.xml

```diff
@@ -133,6 +140,13 @@
 
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.jenkins-ci.tools</groupId>
+				<artifactId>maven-hpi-plugin</artifactId>
+				<configuration>
+					<pluginFirstClassLoader>true</pluginFirstClassLoader>

```

Comment:
This is a pretty bad approach to solving the issue. It may fix the plugin, but other plugins may be affected after that. JIRA Plugin has the similar issue though: https://issues.jenkins-ci.org/browse/JENKINS-48357